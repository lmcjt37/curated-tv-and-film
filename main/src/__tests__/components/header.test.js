import React from 'react';
import ReactDOM from 'react-dom';
import Header from '../../components/header';
import render  from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('snapshot of initial component', () => {
    const component = render.create(<Header />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
