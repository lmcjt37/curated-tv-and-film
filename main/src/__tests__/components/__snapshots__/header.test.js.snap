// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of initial component 1`] = `
<div
  class="Header-root-125"
>
  <header
    class="MuiPaper-root-141 MuiPaper-elevation4-147 MuiAppBar-root-132 MuiAppBar-positionFixed-133 MuiAppBar-colorPrimary-139 mui-fixed"
  >
    <div
      class="MuiToolbar-root-168 MuiToolbar-regular-170 MuiToolbar-gutters-169"
    >
      <h6
        class="MuiTypography-root-172 MuiTypography-h6-189 MuiTypography-colorInherit-201 MuiTypography-noWrap-198 Header-title-127"
      >
        Curated TV and Films
      </h6>
      <div
        class="Header-search-128"
      >
        <div
          class="Header-searchIcon-129"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root-207"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </div>
        <div
          class="MuiInputBase-root-216 Header-inputRoot-130 MuiInputBase-fullWidth-225"
        >
          <input
            class="MuiInputBase-input-226 Header-inputInput-131"
            placeholder="Searchâ€¦"
            type="text"
          />
        </div>
      </div>
      <div
        class="Header-grow-126"
      />
      <button
        class="MuiButtonBase-root-239 MuiIconButton-root-233 MuiIconButton-colorInherit-234"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiIconButton-label-238"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root-207"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <path
              d="M3 17v2h6v-2H3zM3 5v2h10V5H3zm10 16v-2h8v-2h-8v-2h-2v6h2zM7 9v2H3v2h4v2h2V9H7zm14 4v-2H11v2h10zm-6-4h2V7h4V5h-4V3h-2v6z"
            />
          </svg>
        </span>
      </button>
    </div>
  </header>
</div>
`;
